hack/ci/windows.ps1

```diff
@@ -602,10 +601,10 @@ Try {
     }
 
     # Arguments: Allow setting optional storage-driver options
-    # example usage: DOCKER_STORAGE_OPTS="lcow.globalmode=false,lcow.kernel=kernel.efi"
+    # example usage: DDOCKER_STORAGE_OPTS="size=40G"
     if (-not ("$env:DOCKER_STORAGE_OPTS" -eq "")) {
         Write-Host -ForegroundColor Green "INFO: Running the daemon under test with storage-driver options ${env:DOCKER_STORAGE_OPTS}"
-        $env:DOCKER_STORAGE_OPTS.Split(",") | ForEach {
+        $env:DOCKER_STORAGE_OPTS.Split(",") | ForEach-Object {
```

Comment:
Why the change?

Reply:
Linter complaining that Foreach is an alias, and that should not be used as it could lead to unpredictable behavior